alerts/google-gce/metadata.yaml

```diff
@@ -0,0 +1,16 @@
+alert_policy_templates:
+-
+  id: cpu-utilization-too-high
+  display_name: cpu utilization too high
```

Comment:
please match with the displayName in actual policies. Apply to other ones.

Reply:
Display names in metadata.yaml now matches the display names in the individual json files.